backend/src/main/scala/com/softwaremill/bootzooka/user/domain/User.scala

```diff
@@ -19,28 +17,18 @@ case class User(
 )
 
 object User {
+  val Iterations  = 2
```

Comment:
To powinno być z configa?

Reply:
Jakiś stary commit patrzyłeś, potem została wprowadzona konfiguracja

Reply:
Patrzę na "files changed", a nie konkretne commity..,

Reply:
My mistake, removed obsolete configuration params from this object